analysis/monkbrewmaster/src/normalizers/ExpelHarm.ts

```diff
@@ -0,0 +1,45 @@
+import SPELLS from 'common/SPELLS';
+import EventOrderNormalizer from 'parser/core/EventOrderNormalizer';
+import { EventOrder } from 'parser/core/EventOrderNormalizer';
+import { EventType } from 'parser/core/Events';
+import { Options } from 'parser/core/Module';
+
+import { GIFT_OF_THE_OX_SPELL_IDS } from '../constants';
+
+const CelestialFortuneHealOrder: EventOrder = {
+  beforeEventId: SPELLS.EXPEL_HARM.id,
+  beforeEventType: EventType.Cast,
+  afterEventId: SPELLS.CELESTIAL_FORTUNE_HEAL.id,
+  afterEventType: EventType.Heal,
+  bufferMs: 50,
+  anyTarget: true,
+  updateTimestamp: true,
+};
+
+const ExpelHarmHealOrder: EventOrder = {
+  beforeEventId: SPELLS.EXPEL_HARM.id,
+  beforeEventType: EventType.Cast,
+  afterEventId: SPELLS.EXPEL_HARM.id,
+  afterEventType: EventType.Heal,
+  bufferMs: 50,
+  anyTarget: true,
+  updateTimestamp: true,
+};
+
+export default class ExpelHarm extends EventOrderNormalizer {
+  constructor(options: Options) {
+    super(options, [
+      CelestialFortuneHealOrder,
+      ExpelHarmHealOrder,
+      ...GIFT_OF_THE_OX_SPELL_IDS.map((id) => ({
+        beforeEventId: SPELLS.EXPEL_HARM.id,
+        beforeEventType: EventType.Cast,
+        afterEventId: id,
+        afterEventType: EventType.Heal,
+        bufferMs: 50,
+        anyTarget: true,
+        updateTimestamp: true,
```

Comment:
Instead of doing this, you should pass the array of spellIds into `afterEventId` (which supports arrays). I think for performance reasons its preferable to use an array in one EventOrder rather than multiple EventOrder specs.

Reply:
I merged the gift of the ox ids into one event order. I also tried to add the other spells into one event order, but somehow, it would not work, I did not dig too much :man_shrugging: